### PR TITLE
Get remove of ez-stream.

### PR DESCRIPTION
**Pull request by xingwu1**

### CODE REVIEW

**Comment by yugangw-msft** [comment]
@xingwu1 thank you!

**Comment by amarzavery** [comment]
@xingwu1 - The CI is running from 30 minutes
Please take a look at this job: https://travis-ci.org/Azure/azure-xplat-cli/jobs/119073913#L1841

**Comment by amarzavery** [comment]
@xingwu1 - Please use the -q switch in all your tests wherever applicable
https://github.com/Azure/azure-xplat-cli/blob/dev/test/commands/batch/cli.batch.file-tests.js#L208

File: lib/commands/batch/batch.node._js [hunk 1]
```diff
@@ -669,7 +669,9 @@ exports.init = function(cli) {
     var force = !!options.quiet;
     if (utils.fileExists(rdpFile, _)) {
       if (force !== true) {
+        console.log(force);
```
Comment (amarzavery) [inline lib/commands/batch/batch.node._js]: please remove the console.log statements.

**Comment by amarzavery** [comment]
@xingwu1 - Still the same issue. CI has been running for more than 34 minutes https://travis-ci.org/Azure/azure-xplat-cli/jobs/119088891#L1846

**Comment by xingwu1** [comment]
@amarzavery the test cases passed.

**Pull request merged**